### PR TITLE
proxy: simplify channel receive, add missing function call

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

File: proxy/director.go [hunk 1]
```diff
@@ -56,10 +56,8 @@ func newDirector(urlsFunc GetProxyURLs, failureWait time.Duration, refreshInterv
 					plog.Infof("endpoints found %q", sl)
 				})
 			}
-			select {
-			case <-time.After(ri):
-				d.refresh()
-			}
+			<-time.After(ri)
```
Comment (heyitsanthony) [inline proxy/director.go]: `time.Sleep`?

**Comment by heyitsanthony** [comment]
lgtm once it's using `Sleep`

was this from go-simple?

File: proxy/director.go (hunk 1 above)
Comment (gyuho) [inline proxy/director.go]: just fixed. PTAL. Thanks.

**Comment by gyuho** [comment]
@heyitsanthony Yeah it's from go-simple and unused. :) Merging in greens.

**Comment by gyuho** [comment]
CI keeps failing from https://github.com/coreos/etcd/issues/4990. Will investigate more in that issue.

**Pull request merged**